### ISSUE TITLE
VAC Error: 'Could Not Verify Your Game Session' on One Account Only (Steam Family Sharing & Invalid Signatures)

### ISSUE THREAD

**Issue by benbuch** [ValveSoftware/steam-for-linux#11520]
#### Your system information

* Steam client version (build number or date): 1732590255, Tue, Nov 26 1:12 AM UTC -08:00
* Distribution (e.g. Ubuntu): Arch
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes
* Steam Logs: [Gen
[steam-logs.tar.gz](https://github.com/user-attachments/files/17929552/steam-logs.tar.gz)
erate by running this command in a terminal tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs]
* GPU: AMD Radeon RX 6600

#### Please describe your issue in as much detail as possible:

- I can play CS2 fine on my account, but when switching to my son's account, the error persists.
- The error "Some of your game files have been detected to have no signatures or invalid signatures. You will not be allowed to join VAC secure servers." appears when trying to launch the game using Proton.
- This issue only occurs on my system, and the game works fine when my son plays on a different computer.

Background:

- My son encountered a system shutdown due to Steam Family Sharing, which closed his game while he was actively playing. This may have corrupted something in his account or on the server side.
- I have tried a full uninstallation and reinstallation of Steam, and followed common troubleshooting steps (verify integrity, delete Steam cache, etc.) without resolving the issue.

I believe this could be related to Steam Family Sharing or possibly a server-side issue affecting his account.

#### Steps for reproducing this issue:

1. Start CS2 using my son’s account.
2. Try to join any online match.
3. Encounter the "VAC could not verify your game session" error message.
4. (Optional) Try launching the game with Proton, which results in the invalid or missing signatures error message.

**Comment by draugfein** [comment]
Similar issue to mine. (Issue before yours) I am installing it on the flatpack version... Let's hope for the best! :D

**Comment by draugfein** [comment]
Yep, using the flatpack version of the steam client fixed it for me, there is some issue with some amd package....
